### PR TITLE
Added French Translation 

### PR DESCRIPTION
**Pull request by roobini-gamer**
I love this program 
so I decided to add languages I know
I was planning to add in Arabic too but since it is written from right to left I decided not to
I'm no french person but This French Translation is like 98% accurate any french user will be able to understand it easy

### CODE REVIEW

**Comment by DevilXD** [comment]
Hello o/

Looks good! The slightly inaccurate translation is okay, anyone interested will be able to PR changes to it later if needed. 

Can you add yourself to the README file's credits section at the very end too, by chance? If not, I can also add you myself after the merge.

**Comment by roobini-gamer** [comment]
credit added 🌷

File: README.md [hunk 1]
```diff
@@ -71,3 +71,4 @@ if they aren't already there. Doing so ensures proper markdown rendering on Gith
 @ThisIsCyreX - For the entirety of the German (Deutsch) translation.  
 @zarigata - For the entirety of the Portuguese (Português) translation.  
 @alikdb - For the entirety of the Turkish (Türkçe) translation.
```
Comment (DevilXD) [inline README.md]:
```suggestion
@alikdb - For the entirety of the Turkish (Türkçe) translation.  
```

**Review by DevilXD — COMMENTED** [review]
The previous line needs to have two spaces added for proper Markdown formatting on Github, per the comment left in the file. You can just commit my suggestion to fix this.

**Comment by DevilXD** [comment]
Looks good now! Thank you for your contribution =)

**Pull request merged**